orttraining/orttraining/python/training/orttrainer.py

```diff
@@ -140,7 +147,13 @@ def __init__(self, model, model_desc, optim_config, loss_fn=None, options=None):
 
         self.model_desc = _ORTTrainerModelDesc(model_desc)
         self.optim_config = optim_config
-        self.options = ORTTrainerOptions(options)
+        
+        if options:
+            self.options = ORTTrainerOptions(options)
+        else:
+            self.options = ORTTrainerOptions()
+        self._enable_internal_postprocess=True
```

Comment:
Not needed: It is on `ORTTrainer.options._internal_use.enable_internal_postprocess`